config/custom-environment-variables.yaml

```diff
@@ -112,3 +112,7 @@ ecosystem:
     store: ECOSYSTEM_STORE
     # Badge service (needs to add a status and color)
     badges: ECOSYSTEM_BADGES
+    # List of module names, or objects { name, config } for instantiation to use in sd-setup
+    setup: SD_SETUP_PLUGINS
```

Comment:
Based on this, anything that's set to that environment variable will override the current `setup`. This means that the `sd-scm` will be overridden. 

I would have expected something closer to an append, rather than a complete overwrite.